channels/src/main/java/com/softwaremill/jox/Channel.java

```diff
@@ -24,9 +24,9 @@
  * - buffered channels, where a given number of sent values might be buffered, before subsequent `send`s block
  * - unlimited channels, where an unlimited number of values might be buffered, hence `send` never blocks
  * <p>
- * The no-argument {@link Channel} constructor creates a rendezvous channel, while a buffered channel can be created
- * by providing a positive integer to the constructor. A rendezvous channel behaves like a buffered channel with
- * buffer size 0. An unlimited channel can be created using {@link Channel#newUnlimitedChannel()}.
+ * To create a channel, use {@link Channel#newRendezvousChannel()}, {@link Channel#newBufferedChannel(int)} and
+ * {@link Channel#newUnlimitedChannel()} methods. Additionally, {@link Channel#newBufferedDefaultChannel()} creates
+ * a buffered channel with a "default" capacity of 16, which should be a good starting point for most use-cases.
```

Comment:
Don't we want to mention here also `withScopedBufferSize` ?

Reply:
That's a follow-up PR I want to make - moving this to Flow (I'll explain there why :) )